sites/mosul/configs/openmrs/initializer_config/ampathformstranslations/F34-mhGAP_Closure_v2_translations_ar.json

```diff
@@ -58,13 +56,10 @@
     "MSF health facility": "مركز صحي تابع لمنظمة أطباء بلا حدود",
     "MSF psychiatric care": "خدمات الصحة النفسية لدى منظمة أطباء بلا حدود",
     "Main location of intervention": "موقع التداخل",
-    "Main location of intervention - If Health Facility, specify": "اذا منشأة صحية، حدد",
-    "Main location of intervention - If MSF Health Facility, specify": "اذا منشأة صحية لأطباء بلا حدود، حدد",
-    "Main location of intervention - If Mobile Clinic, specify": "اذا عيادات متنقلة، حدد",
-    "Main location of intervention - If other, specify": "اذا أخرى، حدد",
     "Main type of consultation": "نوع الاستشارة الرئيسي",
     "Main type of intervention": "نوع الاستجابة الرئيسي",
     "Medical care (MSF, MoH, NGO)": "العناية الطبية (منظمة أطباء بلا حدود، أو وزارة الصحة، أو منظمة غير حكومية)",
+    "mhGAP closure form": "نموذج إغلاق mhGAP",
```

Comment:
Added the support for adding the page label translations to the `converter.py` script.

Thanks for the catch